ksqldb-engine/src/test/java/io/confluent/ksql/function/udaf/offset/EarliestByOffsetUdafTest.java

```diff
@@ -108,6 +194,26 @@ public void shouldComputeEarliestLong() {
     // Then:
     assertThat(res.getInt64(VAL_FIELD), is(321L));
   }
+  
+  @Test
+  public void shouldComputeEarliest2Longs() {
```

Comment:
```suggestion
  public void shouldComputeEarliestLongs() {
```

Same for other new methods.